File: esercizi_java/test/compito_python/cliente.py
from Noleggio import Noleggio


class cliente:
    def __init__(self, nome, cognome):
        self.nome = nome
        self.cognome = cognome
        self.veicolo = []

    def addVeic(self, v, giorni):
        self.veicolo.append(Noleggio(v, giorni))

    def getmassimo(self, numero):
        massimo = 0
        variabile = 0
        for n in self.veicolo:
            variabile = n.codice
            if variabile == numero:
                massimo = massimo + n.v.cost*n.n
        return massimo

    def costonoleggio(self):
        costo = 0
        for n in self.veicolo:
            costo = n.v.cost*n.n+costo
        return costo
    
    def ciao(self):
        c = ""
        for n in self.veicolo:
            c = c + n.toString()
        return c

    def toString(self):
        return self.nome + "\t" + self.cognome + "\t" + str(self.costonoleggio()) + "\t["+self.ciao()+"]"

File: esercizi_java/NetBeansProjects/ciaone/src/gestione/cliente.java
package gestione;
import java.util.List;
import java.util.LinkedList;

/**
 *
 * @author tobiasacchetto
 */
public class cliente {
    protected String nome,cognome;
    protected List<Noleggio> veicolo;
    
    public cliente(String nome,String cognome) {
    this.nome = nome;
    this.cognome = cognome;
    this.veicolo = new LinkedList<Noleggio>();
    }
    
    
    public void addVeic(veicolo v, int giorni){
    veicolo.add(new Noleggio(v, giorni));
    }
    
    
    
    public double getmassimo(int numero){
    double massimo = 0;
    int variabile=0;
    for (Noleggio n : veicolo){
        variabile = n.codice;
        if(variabile == numero)
        {
            massimo = massimo+n.v.cost*n.n;
        }}
    return massimo;
    }
    
    public double costonoleggio(){
    double costo=0;
    for (Noleggio n : veicolo)
        costo= n.v.cost * n.n+ costo;
    
    return costo;
    }
    
    public String toString(){
        return nome + "\t"+ cognome +"\t" +costonoleggio()+"\t["+ veicolo.toString()+"]";
    }
    
}
